clients/admin-ui/src/features/common/utils.ts

```diff
@@ -47,3 +47,13 @@ export const getFileNameFromContentDisposition = (
   const match = contentDisposition.match(/filename=(.+)/);
   return match ? match[1] : defaultName;
 };
+
+export const getQueryParamsFromArray = (
+  strings: string[],
```

Comment:
can we re-name `strings` to something more specific, and also add a docstring to this fn?